packages/functional-tests/pages/login.ts

```diff
@@ -78,6 +69,42 @@ type FirstSignUpOptions = {
 export class LoginPage extends BaseLayout {
   readonly path = '';
 
+  get CWTSEngineHeader() {
+    return this.page.getByRole('heading', { name: 'Choose what to sync' });
+  }
+
+  get CWTSEngineBookmarks() {
+    return this.page.getByText('Bookmarks', { exact: true });
```

Comment:
To be a little more specific, you should be able to use `getByLabel` on the engines.

Reply:
https://github.com/mozilla/fxa/pull/16584/commits/c5ef64737087a75098759448e89ea2c02f4ed46b